arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/Schedule.kt

```diff
@@ -519,41 +625,100 @@ sealed class Schedule<Input, Output> {
   /**
    * A single decision. Contains the decision to continue, the delay, the new state and the (lazy) result of a Schedule.
    */
-  data class Decision<out A, out B>(val cont: Boolean, val delay: Duration, val state: A, val finish: Eval<B>) {
+  data class Decision<out A, out B>(val cont: Boolean, val delayInNanos: Double, val state: A, val finish: Eval<B>) {
+
+    constructor(cont: Boolean, delay: Duration, state: A, finish: Eval<B>) : this(cont, delay.nanoseconds.toDouble(), state, finish)
+
+    @Deprecated(
+      "Arrow Fx Duration is deprecated use Double or kotlin.time.Duration",
```

Comment:
I think the message should indicate to use an alternative property, not a type

```suggestion
      "Arrow Fx Duration is deprecated, use duration or delayInNanos",
```

or alternatively follow the same structure as previous deprecated messages